client/my-sites/domains/domain-management/list/style.scss

```diff
@@ -111,193 +85,6 @@
 	}
 }
 
-.domain-item,
-.list-header {
```

Comment:
Both `DomainItem` and `ListHeader` were removed, so these classes are unused and can be removed too